### ISSUE TITLE
Implicit discardable tags

### ISSUE THREAD

**Issue by Zaczero** [openstreetmap-ng/openstreetmap-ng#110]
When processing new changes, we want to automatically discard certain tags so clients don't have too. JOSM and iD already implement such features but we should apply it globally for simplicity.

The goal is to:
- efficiently discard certain tags during tagsvalidating validation function (it should return modified dict of tags), see models/validating/tags
- migrate rules from JOSM and iD

I think if the discard method is simple enough (which is very likely), it should be a part od the validating model file.

**Comment by starsep** [comment]
More info at [OSM Wiki](https://wiki.openstreetmap.org/wiki/Discardable_tags) including references to iD/JOSM

**Comment by docentYT** [comment]
Is it enough to modify the tags instance passed as a parameter to the `validate_tags` method, or do we need to make a copy of all the tags?

**Comment by Zaczero** [comment]
It's okay to modify it inplace

**Comment by docentYT** [comment]
I found that the same validation also takes place for changesets, which may include tags such as `created_by`.

**Comment by Zaczero** [comment]
Good catch, let's focus on validating elements only, then. Maybe it’s possible to do this within `ElementValidating`?
